src/columnsTypes/types/Label/utils.js

```diff
@@ -0,0 +1,161 @@
+import * as nacl from 'tweetnacl';
+import {encode as decodeUTF8} from '@stablelib/utf8';
+import {encode as encodeBase64} from '@stablelib/base64';
+import {findParentNodeOfTypeClosestToPos} from 'prosemirror-utils';
+import {PluginKey} from 'prosemirror-state';
+import {
+  EDITOR_LEFT_OFFSET,
+  EDITOR_TOP_OFFSET,
+} from '../../../headers/headers-menu/utils';
+
+export const tableLabelsMenuKey = new PluginKey('TableLabelsMenu');
+
+export const generateHash = (value) => {
+  const buffer = decodeUTF8(value);
+  const hashed = nacl.hash(buffer);
+  const hashValue = encodeBase64(hashed);
+  return hashValue;
+};
+
+export const stringToHash = (strToHash) => {
+  // randomize so similar words have different colors
+  let hash = 0;
+  if (strToHash.length === 0) {
+    return hash;
+  }
+  for (let i = 0; i < strToHash.length; i += 1) {
+    const char = strToHash.charCodeAt(i);
+    // eslint-disable-next-line no-bitwise
+    hash = (hash << 5) - hash + char;
+    // eslint-disable-next-line no-bitwise
+    hash &= hash; // Convert to 32bit integer
+  }
+  return hash;
+};
```

Comment:
I don't think this function needs to be exported. Also, the comment at the top of this function is misleading. This function doesn't randomize, or have anything to with color.